### ISSUE TITLE
DJI V2 goggles and AU 1 no audio

### ISSUE THREAD

**Issue by dang-itt** [funneld/djifpv_live_audio_V2#1]
Hi, I've installed the v05 packages in both the AU and goggles v2. I'm not getting any audio.

**Comment by KTM1551** [comment]
I have the same issue, but it records to my SD Card if its inserted it wont work to the goggles, recording or not though..

SO.. I've figured out you can get the Live Audio to work on the V2's with these .IPK's if you just completely remove the SD Card from the Air Unit before power up, and you will get live Audio to the Goggles through the Headphone port. It doesn't record on the DVR Video file, and you cant record that nice 1080p60 AND have live audio.

So it seems like Live Audio and no recording on the Air Unit, OR you can have Audio recorded to the SD Card's videos..

I cant get it do both no matter how i install the packages, set the volume through CLI, etc. The Live Audio does indeed work but only because i happened to try it first without my SD Card by accident and then realized the second flight, it was the SD being physically inserted that caused the issues for me. I think i even made sure to stop the auto record on take off to try that and see if it worked when not recording, but it doesnt. 

So basically i got lucky and figured this out by pure chance, and since i almost always run an SD Card im sure i would of just given up after a while because no one has OG Air units anymore and no one seemed to care that much about this feature or didnt even know it existed the whole time..

**Comment by mariansoban** [comment]
I can confirm that if I remove SD card from Air Unit, audio-out starts to work when IPK's are installed. It even looks like that there is no need to have recording running.
One more note to installation process: I only could install air-side IPK when there was already system package _wtfos-modloader 0.2.2_ installed  in the unit - e.g. via package manager in browser.
